php/aboutMeFunctions.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+* This Function Fetches Data from the database and returns the query as an associative array.
+*
+* @param the $db is to input the new instance of PDO database.
```

Comment:
@param missing datatype before variable instead of 'the'

Reply:
done